src/cli/telemetry/telemetry.ts

```diff
@@ -143,27 +144,29 @@ export const prepareData = async (
   const build = coreCompiler.buildId || 'unknown';
   const has_app_pwa_config = hasAppTarget(config);
   const anonymizedConfig = anonymizeConfigForTelemetry(config);
+  const isBrowserEnv = IS_BROWSER_ENV;
 
   return {
-    yarn,
-    duration_ms,
+    arguments: config.flags.args,
+    build,
     component_count,
-    targets,
+    config: anonymizedConfig,
+    cpu_model,
+    duration_ms,
+    has_app_pwa_config,
+    isBrowserEnv,
```

Comment:
can we keep this as `snake_case` for consistency with the rest of the properties?

I don't recall off the top of my head _why_ it needs to be like this. I know we don't have any adapter b/w this and the data store (so it gets stored literally as you see it here). That alone isn't much of an issue, unless there's something in the serialization I'm not remembering 🤔